IB Diploma/student_manage_add.php

```diff
@@ -35,117 +38,37 @@
     if (isset($_GET['return'])) {
         returnProcess($guid, $_GET['return'], null, null);
     }
+	
+	$form = Form::create('action', $_SESSION[$guid]['absoluteURL'].'/modules/'.$_SESSION[$guid]['module'].'/student_manage_addProcess.php');
+	$form->setFactory(DatabaseFormFactory::create($pdo));
+	$form->addHiddenValue('address', $_SESSION[$guid]['address']);
+	$form->setClass('smallIntBorder fullWidth');
+	
+	$row = $form->addRow();
+		$row->addLabel('gibbonPersonID', __('Students'));
+		$row->addSelectStudent('gibbonPersonID',$_SESSION[$guid]['gibbonSchoolYearID'], array('allStudents' => true, 'byName' => false, 'byRoll' => true, 'showRoll' => true))->selectMultiple()->isRequired();
+		
+	$data = array('gibbonSchoolYearID' => $_SESSION[$guid]['gibbonSchoolYearID']);
+	$sql = "SELECT gibbonSchoolYearID as value,name FROM gibbonSchoolYear ORDER BY sequenceNumber";
+	$row = $form->addRow();
+		$row->addLabel('gibbonSchoolYearIDStart', __('Start Year'));
+		$row->addSelect('gibbonSchoolYearIDStart')->fromQuery($pdo, $sql)->placeholder(__('Please select...'))->isRequired();
```

Comment:
Luckily the Select class will auto-add the `__('Please select...')` placeholders if the field is required. Helps cut down on a bit of code, and also helps if the text ever changes in the future.